var/spack/repos/builtin/packages/aluminum/package.py

```diff
@@ -62,8 +67,13 @@ class Aluminum(CMakePackage, CudaPackage, ROCmPackage):
     depends_on("cub", when="@:0.1,0.6.0: +cuda ^cuda@:10")
     depends_on("hipcub", when="@:0.1,0.6.0: +rocm")
 
+    depends_on("rccl", when="+rccl")
```

Comment:
```suggestion
    depends_on("rccl", when="+rccl~libfabric")
```
I'm assuming `rccl` and `aws-ofi-rccl` are mutually exclusive?

If that's the case, it might be better to model this with a virtual, and rccl and aws-ofi-rccl both provide it.

Reply:
> I'm assuming rccl and aws-ofi-rccl are mutually exclusive?

`aws-ofi-rccl` depends on definitions in `rccl`... providing a dynamically loadable network plugin for the latter.

Reply:
@becker33 No they are not mutually exclusive.  The `aws-ofi-rccl` package is a plugin to `rccl` on certain systems that can accelerate the `rccl` calls.  Note that not all systems would want it.